pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -810,6 +810,142 @@ func (m *kubeGenericRuntimeManager) updatePodContainerResources(pod *v1.Pod, res
 	return nil
 }
 
+// computeInitContainersAction computes the actions need to take for init
```

Comment:
it is actually much more readable like you did, but reviewing the differences is difficult since the method location has changed

Reply:
moved it to kuberuntime_container.go